README.md

```diff
@@ -1 +1,26 @@
 # Resonate-Backend
+
+With the rising popularity of social voice platforms such as Clubhouse and Twitter Spaces, it is high time for an Open Source alternative. A platform like this would not only enhance credibility within the open-source community, but also attract more users and foster growth. An engagement platform that is Open Source has the potential to drive significant traction and help establish a strong presence.
+
+## Installation
```

Comment:
Can we change it to any other phrase. Installation is not the right work I guess. Something like Setting Up The Environment ?

Reply:
thanks for pointing out.